src/monodroid/jni/monodroid-glue.c

```diff
@@ -100,6 +101,22 @@ static int wait_for_gdb;
 static volatile int monodroid_gdb_wait = TRUE;
 static int android_api_level = 0;
 
+// Values correspond to the CPU_KIND_* macros
+static char* android_abi_names[CPU_KIND_X86_64+1] = {
```

Comment:
I'm kinda surprised that *not* using `const char` doesn't result in a warning from the compiler.  (Double negatives for the win?)